lib/datasource/crate/index.ts

```diff
@@ -161,12 +162,10 @@ export class CrateDatasource extends Datasource {
    */
   private static async fetchRegistryInfo({
     lookupName,
-    registryUrl,
+    registryUrl = '',
```

Comment:
looks like `registryUrl` is never `null` or `undefined`

Reply:
I think we should use `crate.getReleases` instead of common `datasource.getPkgReleases` here and with further datasources. What's the reason behind importing `getPkgReleases` in order to test particular datasource?

Reply:
I think it's to make sure the new datasource is registered. 🤷‍♂️

Reply:
Okay, then I'm gonna to test this explicitly in `datasource/index.spec.ts`. Otherwise, coverage and strict nulls may complicate things for many other datasources.

Reply:
I think the datasource index also does some validation too, so it's convenient extra validation for each datasource